create_project.sh

```diff
@@ -0,0 +1,10 @@
+openstack project create test
+openstack project create --parent test test-subproject
+
+openstack user create --project test --password test-password test
+openstack user create --project test-subproject --password test-subproject-password test-subproject
+
+openstack role add --user test --project test owner
+openstack role add --user test --project test member
+openstack role add --user test --project test-subproject member
+openstack role add --user test-subproject --project test-subproject lessee
```

Comment:
Can you group the non-subproject commands together, and the subproject commands together? That makes it far easier to copy and paste, assuming that someone has no desire to create a subproject. Also add some comments explaining what the commands are doing?